config/src/main/java/org/springframework/security/config/annotation/web/configurers/HeadersConfigurer.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
`featurePolicy()` has not been updated to allow lambda configuration.
This is because there is no reasonable default for the `policyDirectives` argument.
Instead, it can be the last configuration for `headers()`.
```
http
				.headers(headers ->
					headers
						.defaultsDisabled()
						.referrerPolicy(referrerPolicy ->
								referrerPolicy.policy(ReferrerPolicy.SAME_ORIGIN)
						)
						.featurePolicy("")
				);
```